pkgs/os-specific/linux/ippcrypto/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib,
+  stdenvNoCC,
+  fetchurl,
+  fetchFromGitHub,
+  autoconf,
+  automake,
+  binutils,
+  bison,
+  cmake,
+  file,
+  flex,
+  gcc,
+  git,
+  gnumake,
+  gnum4,
+  libtool,
+  nasm,
+  ocaml,
+  ocamlPackages,
+  openssl,
+  perl,
+  python3,
+  texinfo
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "ippcrypto";
+  version = "ippcp_2020u3";
+  src = fetchFromGitHub {
+    owner = "sbellem";
+    repo = "linux-sgx";
+    rev = "e6036d6edb4371f2acc64c50b7cb51e9dfa439a4";
+    sha256 = "0znallianv3lp3y62cfdgp8gacpw516qg8cjxhz8bj5lv5qghchk";
+    fetchSubmodules = true;
```

Comment:
Please devendor the libs.

Reply:
@SuperSandro2000 I am not exactly sure on how to go about to do this. Just to make sure I understand, do you mean that we should not rely on using the dependencies specified in [`.gitmodules`](https://github.com/intel/linux-sgx/blob/master/.gitmodules), but instead fetch these dependencies from `nixpkgs`?

Concretely speaking, there are 4 submodules, and 2 of them are by Intel (dcap & ipp-crypto), meanwhile the other 2 are [`openmp`](https://github.com/NixOS/nixpkgs/blob/3b6c3bee9174dfe56fd0e586449457467abe7116/pkgs/development/compilers/llvm/8/openmp/default.nix) (not sure about the version) and [`oneDNN`](https://github.com/NixOS/nixpkgs/blob/3b6c3bee9174dfe56fd0e586449457467abe7116/pkgs/development/libraries/oneDNN/default.nix).  In both cases, (`openmp` and `onednn`), Intel builds and patches them in their own "custom" way, with an external Makefile maintained under the linux-sgx repo, which itself depends on a `buildenv.mk` file maintained under the same repo.

Not sure if this information helps, and is relevant, but if you could provide a bit of details about what "devendoring" would mean in this context, that would be helpful. Thanks!